### PR TITLE
[feat] added login and signup pages

### PR DESCRIPTION
**Pull request by aryan-13**
Added the following: 
- SIgnup Page
- Login Page

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ import HomePage from './pages/HomePage';
 import ProductListing from './pages/ProductListing';
 import CartPage from './pages/CartPage';
 import WishlistPage from './pages/WishlistPage';
+import Login from './pages/Login';
```
Comment (anshikaa12) [inline src/App.jsx]: can make an index page and import from there looks more clean

**Review by anshikaa12 — APPROVED** [review]
left a small suggestion , everything else looks great

**Review by lannisterrr — APPROVED** [review]
Code looks fine other than points mentioned by anshika.

File: src/App.jsx (hunk 1 above)
Comment (aryan-13) [reply src/App.jsx]: Okay, I'll make the changes!

**Pull request merged**